src/components/main.js

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Country from './countrylist';
+import { getCovidFromApi } from '../redux/actions/fetchapi';
+
+// const todayDate = new Date().toISOString().slice(0, 10);
+const Home = () => {
+  const dispatch = useDispatch();
+  const dataCovid = useSelector((state) => state.covidReducer);
+  // console.log(dataCovid, ' new data covid response');
+  useEffect(() => {
+    if (dataCovid) {
+      dispatch(getCovidFromApi());
+      // dispatch(getCovid());
+    }
+  }, []);
+  // const handleSub = () => {
+  //   const num = document.getElementById('date').value;
+  //   const baseUrldate = `https://api.covid19tracking.narrativa.com/api/${num}`;
+  //   if (num > todayDate) {
+  //     alert('Please, provide a valid date!');
+  //     document.getElementById('date').value = '';
+  //   } else {
+  //     const api = () => async () => {
+  //       const request = await fetch(baseUrldate);
+  //       const response = await request.json();
+  //       const data = response.dates[num].countries;
+  //       const entries = Object.entries(data);
+  //       dispatch(getCovid(entries));
+  //     };
+  //     dispatch(api());
+  //   }
+  // };
+  const [value, setValue] = useState('');
+  const inputHandler = (e) => {
+    setValue(e.target.value);
+  };
+  return (
+    <div>
+      <form>
+        {/* <input type="date" id="date" format="YYYY-MM-DD" /> */}
+        {/* <button id="sub" onClick={handleSub} type="button"> */}
+        {/* CLICK! */}
+        {/* </button> */}
```

Comment:
- [ ] Please it would be better to remove the unused commented code in order to keep you file clean and pro